Services/Mail/classes/class.ilFormatMail.php

```diff
@@ -50,7 +50,7 @@ public function formatReplyMessage()
     * @access	public
     * @return string
     */
-    public function formatReplySubject()
+    public function formatReplySubject() : mixed
```

Comment:
The `mixed` type is not supported in PHP 7.4, which we'll have to support as well in ILIAS 8.